### ISSUE TITLE
Cant use the Widgets!

### ISSUE THREAD

**Issue by phibr0** [bprayush/school-ui-toolkit#4]
`import 'package:flutter/material.dart';
import 'package:school_ui_toolkit/school_ui_toolkit.dart';

class HomePage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: ListView(
        children: [
            ProfileCard(
              margin: EdgeInsets.all(8),
              email: 'email',
              name: "name",
              phoneNumber: "n/a",
              post: "post",
              imageURL:
                  'https://gepa-mbh.de/wp-content/uploads/2019/01/blank-profile-picture-973460.png', //Test Picture URL
            ),
        ],
      ),
    );
  }
}`

Basically this gives the Error:

`The method 'setHeight' was called on null.
Receiver: null
Tried calling: setHeight(15)`

**Comment by bprayush** [comment]
Ooops! Thanks for the bug report. Since, the package uses ScreenUtil to scale. An instance of the screen util needs to be created in the main.dart. Fixing this now. :)

**Comment by bprayush** [comment]
Check for v2.4.4:)
